src/components/Text/README.md

```diff
@@ -0,0 +1,206 @@
+<!--GITHUB_BLOCK-->
+
+# Text
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {Text} from '@gravity-ui/uikit';
+```
+
+## Variant
+
+These are the default variants that can be overridden in the project
+
+- body:
+  - 1: font-size: 13px; line-height: 18px; (used by default)
+  - 2: font-size: 15px; line-height: 20px;
+  - 3: font-size: 17px; line-height: 24px;
+  - short: font-size: 13px; line-height: 16px;
+- caption:
+  - 1: font-size: 9px; line-height: 12px;
+  - 2: font-size: 11px; line-height: 16px;
+- header:
+  - 1: font-size: 20px; line-height: 24px; font-weight: 600;
+  - 2: font-size: 24px; line-height: 28px; font-weight: 600;
+- subheader:
+  - 1: font-size: 13px; line-height: 18px; font-weight: 600;
+  - 2: font-size: 15px; line-height: 20px; font-weight: 600;
+  - 2: font-size: 17px; line-height: 24px; font-weight: 600;
+- display:
+  - 1: font-size: 28px; line-height: 36px; font-weight: 600;
+  - 2: font-size: 32px; line-height: 40px; font-weight: 600;
+  - 3: font-size: 40px; line-height: 48px; font-weight: 600;
+  - 4: font-size: 48px; line-height: 52px; font-weight: 600;
+- code:
+  - 1: font-size: 12px; line-height: 18px; font-weight: 400; font-family: var(--yc-font-family-monospace);
+  - 2: font-size: 14px; line-height: 20px; font-weight: 400; font-family: var(--yc-font-family-monospace);
+  - 3: font-size: 16px; line-height: 14px; font-weight: 400; font-family: var(--yc-font-family-monospace);
+  - inline-1: font-size: 12px; line-height: 14px; font-weight: 400; font-family: var(--yc-font-family-monospace);
+  - inline-2: font-size: 14px; line-height: 16px; font-weight: 400; font-family: var(--yc-font-family-monospace);
+  - inline-3: font-size: 16px; line-height: 20px; font-weight: 400; font-family: var(--yc-font-family-monospace);
+
+<!--LANDING_BLOCK
+
+<ExampleBlock
+    code={`
+<Text variant="body-1">some text</Text>
+<Text variant="caption-2">some text</Text>
+<Text variant="display-3">some text</Text>
+`}>
+    <UIKit.Text variant="body-1">some text</UIKit.Text>
+    <UIKit.Text variant="caption-2">some text</UIKit.Text>
+    <UIKit.Text variant="display-3">some text</UIKit.Text>
+</ExampleBlock>
+
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Text variant="body-1">some text</Text>
+<Text variant="caption-2">some text</Text>
+<Text variant="display-3">some text</Text>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Ellipsis
+
+`false` - used by default.
```

Comment:
Let's describe for what `ellipsis` prop need instead of listing its values. Something like that:
> `ellipsis` property allows you to hide overflowing text